scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -841,17 +844,24 @@ run_becktool() {
 
             m0drlog "Running Becktool on remote node"
             # Below statement is for logging purpose
-            echo "Running '(cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs -g $REMOTE_SEG_GEN_ID;)'" >> ${LOG_FILE}
+            echo "Running '(cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE \
+                            -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs \
+                            -g $REMOTE_SEG_GEN_ID -r $MD_DIR/scan_offset;)'" \
+                            >> ${LOG_FILE}
             run_cmd_on_remote_node "bash -s" <<-EOF
-            (cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs -g $REMOTE_SEG_GEN_ID;)
+            (cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE \
+             -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs \
+             -g $REMOTE_SEG_GEN_ID -r $MD_DIR/scan_offset;)
```

Comment:
Can we save this in a variable and then use it in both places?

This will avoid duplicate the same string in two places.